Tests/SubscriptionTests/API/SubscriptionEndpointServiceTests.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SubscriptionEndpointServiceTests.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+@testable import Subscription
+import SubscriptionTestingUtilities
+
+final class SubscriptionEndpointServiceTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testGetSubscriptionSuccessNoCache() async throws {
+        let token = "someToken"
+        let subscription = DDGSubscription(productId: "productID",
+                                           name: "name",
+                                           billingPeriod: .monthly,
+                                           startedAt: Date(timeIntervalSince1970: 1000),
```

Comment:
These dates would eventually become past dates, so maybe use an interval based on today's date?

Reply:
The test was not using the dates but good point, I have changed it with 
'startedAt: Date.yearAgo,'
'expiresOrRenewsAt: Date.aYearFromNow,'